adafruit_button.py

```diff
@@ -59,12 +65,68 @@ class Button(displayio.Group):
     :param selected_outline: Inverts the outline color.
     :param selected_label: Inverts the label color.
     """
+    RECT: int = const(0)
+    ROUNDRECT: int = const(1)
+    SHADOWRECT: int = const(2)
+    SHADOWROUNDRECT: int = const(3)
+
+    def __init__(
+        self,
+        *,
+        x: int,
+        y: int,
+        width: int,
+        height: int,
+        name: Optional[str] = None,
+        style: int = RECT,
+        fill_color: Optional[int] = 0xFFFFFF,
+        outline_color: Optional[int] = 0x0,
+        label: Optional[Label] = None,
+        label_font: Optional[FontProtocol] = None,
+        label_color: Optional[int] = 0x0,
```

Comment:
label_color can have tuple color as well I think. I'll test this one too.

Reply:
This one was wrong though. `label_color` must be an int (hex color). I verified that tuple does not work.